src/KingSystem/System/PlayReportMgr.h

```diff
@@ -43,20 +51,21 @@ KSYS_CHECK_SIZE_NX150(PlayReportMgr, 0x40);
 
 class PlayReport {
 public:
-    PlayReport(const sead::SafeString& event_id, s32 num_entries, sead::Heap* heap) {
+    inline PlayReport(const sead::FixedSafeString<32>& event_id, s32 num_entries,
```

Comment:
Is the `inline` necessary?

Reply:
Yes, it doesn't produce matching code otherwise. This is inlined inside uking::PlayReport's constructor. Since it generates large code, the compiler won't inline it without the specifier